hbase-client/src/main/java/org/apache/hadoop/hbase/client/Admin.java

```diff
@@ -1312,6 +1312,14 @@ default boolean abortProcedure(long procId, boolean mayInterruptIfRunning) throw
    */
   void rollWALWriter(ServerName serverName) throws IOException, FailedLogCloseException;
 
+  /**
+   * Archive the earliest log file.
+   *
+   * @param serverName The servername of the regionserver.
+   * @throws IOException if a remote or network exception occurs
+   */
+  void archiveWAL(ServerName serverName) throws IOException;
```

Comment:
Should method name reflect that the earliest log is archived ?

Reply:
Good point, will fix later. 
Thanks.